install.sh

```diff
@@ -1,15 +1,97 @@
 #!/usr/bin/env sh
 
 sudo apt-get update
-sudo apt-get install -y ccache cmake clang-format python3-pip python3-setuptools libgmp3-dev parallel
-
+sudo apt-get install -y ccache cmake clang clang-format-7 python3-pip python3-setuptools libgmp3-dev g++ parallel
 python3 -m pip install --upgrade wheel
-python3 -m pip install --upgrade conan
 python3 -m pip install --upgrade trio
 
-conan profile new default --detect
-conan profile update settings.compiler.libcxx=libstdc++11 default
+if [ "$1" = "USE_CONAN" ]; then
+  echo "Using Conan build..."
+  echo "Installing Conan..."
+  python3 -m pip install --upgrade conan
+  conan profile new default --detect
+  conan profile update settings.compiler.libcxx=libstdc++11 default
```

Comment:
Are we sure this actually uses the compiler configured in travis?

Reply:
I think so.
WIth clang Conan we also uses libstdc++11 (see [https://docs.conan.io/en/latest/reference/config_files/settings.yml.html](https://docs.conan.io/en/latest/reference/config_files/settings.yml.html)).

Reply:
Yep. It does. I checked it in the travis logs for a successful build on Friday.